R/compile_report.R

```diff
@@ -231,15 +239,30 @@ compile_report <- function(file, quiet = FALSE, factory = getwd(),
   }
 
   dir.create(output_dir, FALSE, TRUE)
-
-  for (file in new_files) {
+  
+  output_files <- vector(length(new_files), mode = "list")
+  for (i in seq_along(new_files)) {
+    file <- new_files[i]
     destination <- file.path(output_dir, file)
     move_file(file, destination)
+    output_files[[i]] <- destination
   }
-
+  
   ## remove empty directories
   to_remove <- file.path(new_dirs)
   file.remove(to_remove)
+  
+  
+  ## ================
+  ## Start log code
+  log_file_path <- file.path(factory, ".compile_log.rds")
+  current_log <- current_compile_log(log_file_path, base_name)
+  env_list <- as.list(environment())
```

Comment:
Shouldn't `compile_init_env` be used here?

Reply:
no because we're also getting some other things from this environment. the `create_log_entry` function gets `compile_init_env` and some other relevant variables that are added along the way from the `environment` at this point. will add this to the annotation as well.

Reply:
Ah okay got you thx